cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -174,6 +174,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         try {
             repository.delete(releaseName_build)
         } catch (_: Exception) {
+            // The deletion might fail if the folder does not exist; this exception is intentionally ignored as the absence of the folder is acceptable.
```

A comment was added within the catch block to clearly indicate that the exception is being ignored because the folder deletion failure is acceptable if the folder does not exist. This makes the code intent explicit and addresses the Sonar rule violation.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)